library/neutron_network.py

```diff
@@ -18,6 +18,8 @@
 
 try:
     from neutronclient.neutron import client
+    from keystoneauth1.identity import v2
```

Comment:
There is a v2 and a v3 module in keystoneauth, wouldn't we want to use the v3 code (https://github.com/openstack/keystoneauth/tree/master/keystoneauth1/identity/v3)  so we can support different domains for some of this stuff ?